server/src/DBService/Exposure/Mutation.ts

```diff
@@ -1,17 +1,9 @@
 import { gql } from "postgraphile";
 
-export const UPDATE_EXPOSURE = gql`
-mutation updateExposure($exposureId: Int!, $data: ExposurePatch!) {
-  updateExposureById(input: {exposurePatch: $data, id: $exposureId}) {
+export const UPDATE_EXPOSURES = gql`
+mutation updateExposures ($inputExposure: JSON!) {
```

Comment:
maybe change to input exposure**s**

Reply:
Why? I mean I can do that but I think the name is clear enought

Reply:
because you update multiple exposures and not one